scanapi/console.py

```diff
@@ -0,0 +1,82 @@
+from rich.console import Console
+
+from scanapi.test_status import TestStatus
+from scanapi.session import session
+
+
+console = Console()
+
+
+def write_results(results):
+    """Print the test results to the console output
+
+    Returns:
+        None
+    """
+    for r in results:
+        for test in r["tests_results"]:
+            if test["status"] is TestStatus.PASSED:
+                console.print(f"[bright_green] [PASSED] [white]{test['name']}")
+            if test["status"] == TestStatus.FAILED:
+                console.print(
+                    f"[bright_red] [FAILED] [white]{test['name']}\n"
+                    f"\t  [bright_red]{test['failure']} is false"
+                )
+    _write_summary()
+
+
+def log_report(uri):
```

Comment:
maybe we could change this method name to something like `print_report_path` or `write_report_path` for consistency. Because I would assume that a `log_report` method would use a `log` object to log something and not a console to print. What do you think?